src/synthesizer/photoionisation/cloudy17.py

```diff
@@ -234,7 +234,7 @@ def create_cloudy_input(
     which will again introduce issues on mass conservation.
     """
 
-    if (abundances.dust_to_metal_ratio > 0) & params["grains"]:
+    if (abundances.dust_to_metal_ratio > 0) & (params["grains"] is not None):
```

Comment:
Can you swap `&` for `and` here.  In Python `&` is a bitwise operator, although it works here it's not (pedantically) not the right operator.

Reply:
Done